<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDistanz</class>
 <widget class="QMainWindow" name="PDistanz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1325</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>5</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="GalaGV" name="graphicsView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="renderHints">
           <set>QPainter::Antialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWebView" name="webView">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QWebView" name="webViewODDB">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWebView" name="webViewChat">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="url">
           <url>
            <string>about:blank</string>
           </url>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_5">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="Option" name="option" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_6">
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <item>
         <widget class="ScoutWidget" name="widget_2" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionLogin"/>
   <addaction name="actionAddtoChat"/>
   <addaction name="actionShowOwn"/>
   <addaction name="separator"/>
   <addaction name="actionLoadLogs"/>
   <addaction name="actionAlly"/>
   <addaction name="separator"/>
   <addaction name="actionGalaScreen"/>
   <addaction name="separator"/>
   <addaction name="actionOD"/>
   <addaction name="separator"/>
   <addaction name="actionODDBtool"/>
   <addaction name="actionShipConfig"/>
   <addaction name="actionPlaniEditor"/>
   <addaction name="actionAllyForum"/>
   <addaction name="separator"/>
   <addaction name="actionChat"/>
   <addaction name="separator"/>
   <addaction name="actionOption"/>
   <addaction name="actionTest"/>
   <addaction name="actionScout"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_6">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>distanzCalc</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_6">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonCalc">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>calc</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonLoadPlanis">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>loadPlanis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string>weiter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>entfernung</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>check</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>planis</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>chat</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTextBrowser" name="chatEdit">
       <property name="palette">
        <palette>
         <active>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="Link">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>170</red>
             <green>255</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="openLinks">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>loadUrl</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditChat"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonChat">
         <property name="text">
          <string>senden</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Liste</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="ShipTreeWidget" name="treeWidgetSchiffe">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>4</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Planet</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Ankunft</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="windowTitle">
    <string>Favoriten</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="FavTreeWidget" name="treeWidgetFav">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="windowTitle">
    <string>FPCalc</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <layout class="QVBoxLayout" name="verticalLayout_8" stretch="0">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="FPPlani" name="widget" native="true">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_7">
   <property name="windowTitle">
    <string>Bauaufträge</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="BauTreeWidget" name="treeWidgetBau">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true"/>
        </property>
       </column>
       <column>
        <property name="text">
         <string>M</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>K</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>W</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>F</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>IP</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>$</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_8">
   <property name="windowTitle">
    <string>Planeten</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_8">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="PlanetenTreeWidget" name="treeWidgetPlaneten">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>Planet</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>E</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>M</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>W</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>K</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>F</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Bau</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Fertig</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Zeit</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5">
   <property name="windowTitle">
    <string>CombatLog</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="QTreeWidget" name="treeWidgetCombatLog">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_9">
   <attribute name="dockWidgetArea">
    <number>4</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_9">
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="GalaGV" name="graphicsView_2">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="renderHints">
        <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
       </property>
       <property name="dragMode">
        <enum>QGraphicsView::ScrollHandDrag</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_10">
   <property name="windowTitle">
    <string>NearSys</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_10">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="margin">
      <number>2</number>
     </property>
     <item>
      <widget class="NearSys" name="widgetNearest" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>14</height>
    </size>
   </property>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>login</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionShowOwn">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>showOwn</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </action>
  <action name="actionAddtoChat">
   <property name="text">
    <string>addtoChat</string>
   </property>
  </action>
  <action name="actionODDBtool">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ODDBtool</string>
   </property>
  </action>
  <action name="actionPlaniEditor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PlaniEditor</string>
   </property>
  </action>
  <action name="actionShipConfig">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>shipConfig</string>
   </property>
  </action>
  <action name="actionAllyForum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>AllyForum</string>
   </property>
  </action>
  <action name="actionChat">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Chat</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionOD">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>OD</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionGalaScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>GalaScreen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionOption">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Option</string>
   </property>
  </action>
  <action name="actionLoadLogs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>loadLogs</string>
   </property>
  </action>
  <action name="actionAlly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ally</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionScout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>scout</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>GalaGV</class>
   <extends>QGraphicsView</extends>
   <header location="global">galagv.h</header>
  </customwidget>
  <customwidget>
   <class>FPPlani</class>
   <extends>QWidget</extends>
   <header location="global">fpplani.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Option</class>
   <extends>QWidget</extends>
   <header location="global">option.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BauTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">bautreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>PlanetenTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">planetentreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>ShipTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header location="global">shiptreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>FavTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>favtreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>NearSys</class>
   <extends>QWidget</extends>
   <header location="global">nearsys.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScoutWidget</class>
   <extends>QWidget</extends>
   <header location="global">scoutwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
